### ISSUE TITLE
A11y_SQLAzureDataStudio_NewServerGroup_AddServerGroup_ColorContrast:The outline of the edit box is  below the ratio 3:1

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9205]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_SQLAzureDataStudioBenchmark;#A11yWCAG2.1;#A11ySev1;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS1.4.11;#ColorContrast;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Tools:Accessibility insights for desktop
MAS References:MA1.4.11

**Pre-requisite:** apply the various themes available for the application.

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over the 'Servers' header and select the 'New Server Group' icon.
4. Launch accessibility insights for desktop and select the button "Contrast"

**Actual:**
On checking the color contrast of the outline of the edit box is below 3:1 value.

**Same issue is observed for all the pages that contain user interface components like edit box and radio buttons, and button (yes/no) when not in focus for the delete account pop-up.**

**Expected:**
The ratio should be 3:1 or greater.

**User Impact:**
Low vision users will face difficulty in viewing the edit field.

Attachment for Reference :
![image](https://user-images.githubusercontent.com/24580570/74806018-dbc32880-530a-11ea-9303-cc1d4da2936b.png)

**Comment by Charles-Gagnon** [comment]
We have high contrast mode for this - which should meet the requirements as per https://www.w3.org/WAI/WCAG21/Techniques/general/G174

**Comment by alanrenmsft** [comment]
lets also ask this in the office hour. if we need to fix it, it is defined here: extensions/theme-carbon/themes/light_carbon.json

**Comment by fsteffi** [comment]
GithubTags:#A11y_SQlADS_Win32;#A11yMAS;

**Comment by Yashi1201** [comment]
Closing the bug as the issue is not repro
Version : 1.42.0-insider(user setup)
![9205_issue fixed](https://user-images.githubusercontent.com/93374043/217489929-8476092a-9790-4dbf-a96e-edecbc56d92e.png)

**GithubTags**:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
